apps/api/src/block/http-schemas/block.schema.ts

```diff
@@ -3,7 +3,7 @@ import { z } from "@hono/zod-openapi";
 const defaultLimit = 20;
 
 export const ListBlocksQuerySchema = z.object({
-  limit: z.coerce.number().min(1).max(100).default(defaultLimit).optional().openapi({
+  limit: z.coerce.number().min(1).max(100).optional().default(defaultLimit).openapi({
```

Comment:
very likely `.optional()` is redundant in this sequence, because the input is optional and the final value is not optional (but `.default()` must go after `.optional()`)